sbpy/spectroscopy/core.py

```diff
@@ -9,41 +9,262 @@
 
 from abc import ABC
 import numpy as np
+import astropy.constants as con
 import astropy.units as u
-import astropy.constants as const
+from astroquery.jplspec import JPLSpec
+from astropy.time import Time
+from astroquery.jplhorizons import Horizons
+from ..activity.gas import photo_timescale
+from astroquery.jplhorizons import conf
+from scipy import interpolate
```

Comment:
scipy import should come after numpy, then astropy, astroquery and local imports.